WalletWasabi.Packager/Program.cs

```diff
@@ -367,10 +384,34 @@ private static async Task PublishAsync()
 					continue;
 				}
 
-				ZipFile.CreateFromDirectory(currentBinDistDirectory, Path.Combine(BinDistDirectory, $"Wasabi-osx-{VersionPrefix}.zip"));
+				var zipFileName = target.Contains("arm64")
+					? $"_WasabiToMac-osxarm64-{VersionPrefix}.zip"
```

Comment:
nit: I believe it's more common in software industry to follow 

`<PackageName>-<version>-<os>-<architecture>` 

Some examples are here: https://github.com/cli/cli/releases/tag/v2.8.0. 

So my suggestion would be to modify:

`_WasabiToMac-osxarm64-{VersionPrefix}.zip`
->
`_WasabiToMac-{VersionPrefix}-osx-arm64.zip` or `_WasabiToMac-{VersionPrefix}-macOS-arm64.zip` as `macOS` is preferred name now and `osx` is obsolete. Or `_WasabiToSign-{VersionPrefix}-macOS-arm64.zip`.

If possible.

Reply:
This is just the transfer file. In MacSignTools it is easier to extract the version number.

Reply:
I changed the naming to macOS and macOS-arm64 in the final packages.